libtenzir/builtins/operators/top_rare.cpp

```diff
@@ -108,7 +108,10 @@ class top_rare_plugin final : public virtual operator_parser_plugin,
     auto summarized = summarize->make(
       {
         inv.self,
-        {summarize_args, std::move(selector).unwrap()},
+        {
+          std::move(selector).unwrap(),
+          summarize_args,
+        },
```

Comment:
I snuck this in as a small change, don't really think that it needs a changelog entry as it is really only noticeable in the interaction with `chart`, which we haven't even ported to TQL2 yet.

Reply:
Yep, makes sense!